project/release.sh

```diff
@@ -190,6 +190,13 @@ release_build() {
 		linux)
 			s3Os=Linux
 			;;
+		windows)
+			s3Os=Windows
+			binary+=".exe"
+			if [ "$latestBase" ]; then
+				latestBase+=".exe"
```

Comment:
minor nit, can we swap the `"` for `'` here and above since we don't need interpolation?